tests/test_command_line_interface.py

```diff
@@ -1,49 +1,68 @@
 """Test cases for the command line interface."""
 from typer.testing import CliRunner
 
-from src.command_line_interface import cli
+from command_line_interface import cli
 
 
 cli_runner = CliRunner()
 
 
 def test_main_class():
     """Test case to see if class argument is correct."""
-    class_result = cli_runner.invoke(cli, ["--class_def"])
+    class_result = cli_runner.invoke(cli, ["--input-file", "tests/input/sample_file.py", "--class_def"]) # pylint: disable=C0301
```

Comment:
The line length lint check should not be disabled; instead, you should decrease the length of this line by splitting it into two or more.

Reply:
@caldwella2